frontend/packages/policy-editor/src/components/PolicyCardRules/PolicyCardRules.test.tsx

```diff
@@ -8,6 +8,8 @@ import {
   PolicyEditorContext,
   type PolicyEditorContextProps,
 } from '../../contexts/PolicyEditorContext';
+import type { PolicyRuleCard } from '@altinn/policy-editor/types';
+import { getPolicyRuleIdString } from '@altinn/policy-editor/utils/PolicyRuleUtils';
```

Comment:
Should use relative import here 😄 
```suggestion
import type { PolicyRuleCard } from '../../types';
import { getPolicyRuleIdString } from '../../utils/PolicyRuleUtils';
```

Reply:
Thanks! My IDE keeps using these imports 😅